ddtrace/internal/_encoding.pyx

```diff
@@ -234,7 +234,7 @@ cdef class MsgpackStringTable(StringTable):
         self._lock = threading.RLock()
         super(MsgpackStringTable, self).__init__()
 
-        assert self.index(ORIGIN_KEY) == 1
+        self.index(ORIGIN_KEY)
```

Comment:
Clarification because I struggle to read this language: 
We are basically switching off caring about the return value of index? Is that ok to do in this instance? Or should we add something like:
``
```suggestion
        ret = self.index(ORIGIN_KEY)
        if re != 1:
            raise Error
```

Reply:
I think the `assert` I originally added was pedantry, since we tested for the expected behaviour. I have improved what tests we already had so that we can remove this safely.

Reply:
Got it!